internal/client/client.go

```diff
@@ -84,18 +84,40 @@ func (c *CSAPI) JoinRoom(t *testing.T, roomIDOrAlias string, serverNames []strin
 	return GetJSONFieldStr(t, body, "room_id")
 }
 
-// SendEventSynced sends `e` into the room and waits for its event ID to come down /sync.
-// Returns the event ID of the sent event.
-func (c *CSAPI) SendEventSynced(t *testing.T, roomID string, e b.Event) string {
+func (c *CSAPI) SendEvent(t *testing.T, roomID string, e b.Event) string {
```

Comment:
Separated `SendEvent` from `SendEventSynced`

We use `SendEvent` because we don't expect these backfilled messages to get synced

Reply:
Please do not add `SendEvent`. I purposefully left this out to prevent people accidentally sending events without waiting for them to be synced. Keep this function as a private function for your test please.